src/WordSearch/WordSearchResult.ts

```diff
@@ -0,0 +1,10 @@
+import {Point} from './WordSearchPoint';
+
+export class WordSearchResult {
+  word: string;
+  points: Point[];
+  constructor(word: string, wordPoints: Point[]) {
+    this.word = word;
+    this.points = wordPoints;

```

Comment:
You probably want a deep copy here to prevent the caller from messing with this array after passing it to you.  I believe that right now this just copies a reference to the array, not the elements.  @kfarnung does that seem correct?

Reply:
That's not a bad idea, it might be too new, but `Array.from(other)` should make a shallow copy of the elements.  I don't think it would be too surprising to most devs that modifying the array after passing it could blow up, but might as well harden the surface and if the copy ever becomes a problem, deal with it later.